front/src/config/i18n/en.json

```diff
@@ -221,6 +222,12 @@
         "noImageToShow": "No image to show",
         "editBoxNameLabel": "Enter the name you want to give to the box:",
         "editBoxNamePlaceholder": "Name of the box"
+      },
+      "health": {
+        "editDeviceFeaturesLabel": "Select the devices you want to display.",
+        "noValue": "No value recorded",
+        "deviceTitle": "{{name}} - {{type}}",
+        "noDevices": "Looks you don't have any devices in your house!<br/>Define your device in the integration page."
```

Comment:
noDevices isn't in MarkupText, is the <br/> well interpreted?

Reply:
yes I use it in <Text> and works fine